spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/AppDeployerReleaseManager.java

```diff
@@ -89,18 +89,19 @@ public Release install(Release releaseInput) {
 		Release release = this.releaseRepository.save(releaseInput);
 		logger.debug("Manifest = " + releaseInput.getManifest());
 		// Deploy the application
-		List<SpringBootAppKind> springBootAppKindList = SpringBootAppKindReader.read(release.getManifest());
+		List<SpringCloudDeployerApplicationKind> springCloudDeployerApplicationKindList = SpringCloudDeployerApplicationKindReader
```

Comment:
Since this is effectively a breaking change, we need to handle this more gracefully.  This code would not work if it attempted to load a skipper m1 package.  Couple options
1. support both old and new package formats in M2
2. read the yml into a map and extract the `kind` value to throw a meaningful error if reading an old package.  This feature will generally be needed anyway to support other `kind`s such as the CF manifest.

I'd rather not do 1.... so favor 2.  Thoughts?

Reply:
yeah, I like the option 2 as well. I will give that a try today.